README.md

```diff
@@ -1 +1,64 @@
 # nextcloud-shibboleth-sync
+
+This repo contains an app to synchronize a list of identity providers with the 
+[user_saml](https://github.com/nextcloud/user_saml) Nextcloud application.
+
+This app is written in pure php without dependency and connect to a postgresql database.
+
+## How to build the image
+
+The docker image will copy the `app` directory and the `CMD` instruction directly executes the script.
+
+To build it, run:
+
+```bash
+$ docker build -t fundocker/nextcloud-shibboleth-sync:{TAG} .
+```
+
+## How to use the image
+
+To run the image, several environment variables should be used:
+
+- `NEXTCLOUD_IDPS_METADATA_URL`: the XML url containing the identity providers list
+- `SAML_DISPLAYNAME_MAPPING`: Attribute to map the displayname to
+- `SAML_EMAIL_MAPPING`: Attribute to map the email address to
+- `SAML_GENERAL_UID_MAPPING`: Attribute to map the UID to
+- `SAML_GROUP_MAPPING`: Attribute to map the users groups to
+- `SAML_SERVICE_PROVIDER_PK`: path to the service provider private key
+- `SAML_SERVICE_PROVIDER_X509_CERT`: path to the service provider X509 certificate
+- `POSTGRES_DB`: postgresql database name
+- `POSTGRES_HOST`: postgresql database host
+- `POSTGRES_PASSWORD`: postgresql database password
+- `POSTGRES_PORT`: postgresql database port
+- `POSTGRES_USER`: postgresql database username
+
+A volume is also used to add the X509 certificate and the private key provided by the identity provider,
+this volume must be mounted in the `/private` directory and the path to each file added to the environment 
```

Comment:
Relative path to `/private` or absolute path?

Reply:
the absolute path `/private`

Reply:
So maybe it's worth to be mentioned explicitly?

Reply:
starting a path with a `/` is not explicit ? I though it was a unix-like convention

Reply:
I am talking about the values of the two environment variables not the volume. Your expression may suggest that environment variable values are relative to the volume mount path.

Reply:
ok done